### ISSUE TITLE
[Feature] Generate clients for the whole collection at once

### ISSUE THREAD

**Issue by nileshtrivedi** [usebruno/bruno#1897]
I intend to start maintaining our API specs in Bruno but will need the ability to generate client code for the whole collection at once. Currently, I use `openapi-generator-cli` like this:

```
openapi-generator-cli generate -i https://server.com/openapi.json -o src/generated-sources/openapi -g typescript-fetch --additional-properties=supportsES6=true,npmVersion=6.9.0,typescriptThreePlus=true
```

**Comment by helloanoop** [comment]
Hey @nileshtrivedi 

Can you explain your use case a bit more.
Is it essentially having an option to convert openapi specs to Bruno via cli as described in this issue #1013 ?

**Comment by nileshtrivedi** [comment]
No, it's the opposite. If my API spec is mentioned in Bruno files, I want to be able to generate client libraries in multiple programming languages from that.
